src/docs/integrations/github.mdx

```diff
@@ -10,26 +10,34 @@ To configure the GitHub integration you'll need to create a GitHub app and obtai
   The GitHub App name must not contain any spaces.
 </Alert>
 
+<CreateGitHubAppForm url="https://github.com/organizations/:org/settings/apps/new?name=:org-Sentry-Integration&public=false&members=read&emails=read&administration=read&contents=read&issues=write&pull_requests=write&repository_hooks=write&url=:url-prefix&callback_url=:url-prefix%2Fauth%2Fsso%2F&setup_url=:url-prefix%2Fextensions%2Fgithub%2Fsetup%2F&webhook_url=:url-prefix%2Fextensions%2Fgithub%2Fwebhook%2F&events[]=push&events[]=pull_request&webhook_active=true" defaultOrg="your-organization" defaultUrlPrefix="https://your-sentry-url-prefix.com" />
```

Comment:
Whoa, really!? 🧐

Reply:
@BYK coming back to haunt us (#143). 😁

Reply:
I just used the same form present in [GitHub SSO docs](https://develop.sentry.dev/self-hosted/sso/#github-auth) page.

Reply:
Reusability <3 me